src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Write your first NRQL alerts condition
+tags:
+  - Alerts and Applied Intelligence
+  - New Relic Alerts
+  - Alert conditions
+  - NRQL
+metaDescription: How to start writing NRQL alerts conditions with a step-by-step user case.
+---
+
+Use our [New Relic query language (NRQL)](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) to create alerts conditions with ease. The query defines the data that's important to you and the condition describes what you want to be notified about.
+
```

Comment:
I'd reorder these two sentences. First, I'd let the user know about the condition. Then, I'd define what the query is for, then I'd go for the threshold.

Your condition describes what you want to be notified about, and only requires two attributes: a query and a threshold. The query defines the data that's important to you and determines what you want to be notified about. The threshold determines when you’ll be notified.

Reply:
Excellent recommendation!